### ISSUE TITLE
When enabling debug, schame validation produces too much output

### ISSUE THREAD

**Issue by jfdenise** [wildfly-extras/wildfly-channel#40]
Context: Maven plugin
Enable debug with "-X" argument: mvn clean package -X
The console displays thousands of lines (channel content).
That is caused by :https://github.com/wildfly-extras/wildfly-channel/blob/main/core/src/main/java/org/wildfly/channel/ChannelMapper.java#L76
That calls:  https://github.com/wildfly-extras/wildfly-channel/blob/main/core/src/main/java/org/wildfly/channel/ChannelMapper.java#L115

This makes debug output unusable.

**Comment by jmesnil** [comment]
Why is it unusable? 

This is caused by the logger `com.networknt.schema.RequiredValidator`.
I'm not sure which actions you would like to make. `mvn -X` will output all debug logs including this one.
Unreadable? For sure! Unusable? Nothing that a grep can not filter out.

**Comment by jfdenise** [comment]
Yes, we could use grep but the amount of redundant output is so huge that it should be trimmed.

**Comment by jmesnil** [comment]
I'm not sure there is much we can do in this project.

We could provide a patch to the project logging `com.networknt.schema.RequiredValidator` to use `TRACE` instead of `DEBUG`...

**Comment by jmesnil** [comment]
The log output can be reduced by adding the sys prop: 

```
-Dorg.slf4j.simpleLogger.log.com.networknt.schema=off
```

when maven is executed: `mvn clean package  -Dorg.slf4j.simpleLogger.log.com.networknt.schema=off -X`

However, I'm not sure if there is a way to generalise this (other than editing the logger configuration used by Maven at ${MAVEN_HOME}/conf/logging/simplelogger.properties

**Comment by jfdenise** [comment]
@jmesnil , should we document the option and keep this behavior?
